src/Components/Admins/Form/index.jsx

```diff
@@ -1,40 +1,115 @@
 import React, { useState, useEffect } from 'react';
 import styles from './form.module.css';
+import { useHistory, useParams } from 'react-router-dom';
+import Button from '../../Shared/Button';
+import ModalAlert from '../../Shared/ModalAlert/index.jsx';
+import { Input } from '../../Shared';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faEye, faEyeSlash } from '@fortawesome/free-solid-svg-icons';
 
-const Form = (props) => {
+const Form = () => {
+  const [modalText, setModalText] = useState('');
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const { id } = useParams();
+  const [adminsData, setAdminsData] = useState([]);
+  const [selectedAdmin, setSelectedAdmin] = useState({});
   const [formData, setFormData] = useState({
-    firstName: '',
-    lastName: '',
-    dni: '',
-    phone: '',
-    email: '',
-    city: '',
-    password: ''
+    firstName: selectedAdmin.firstName || '',
+    lastName: selectedAdmin.lastName || '',
+    dni: selectedAdmin.dni || '',
+    phone: selectedAdmin.phone || '',
+    email: selectedAdmin.email || '',
+    city: selectedAdmin.city || '',
+    password: selectedAdmin.password || ''
   });
+  const closeModal = () => {
```

Comment:
i think this should be setIsModalOpen(!isModalOpen)
as it its rigth now if a alert happens you cant close it